graphql-server/src/utils/servers.ts

```diff
@@ -0,0 +1,68 @@
+import { Express } from "express";
+import path from "path";
+import fs from "fs";
+import https from "https";
+import { buildNeo4jDbSchema } from "../schemas/neo4j";
+import { mergeSchemas } from "@graphql-tools/schema";
+import { oracleDbSchema } from "../schemas/oracle";
+import { ApolloServer } from "apollo-server-express";
+import {
+  ApolloServerPluginDrainHttpServer,
+  ApolloServerPluginLandingPageLocalDefault,
+} from "apollo-server-core";
+import { updateActiveUserSessions } from "./session";
+import { corsOptions } from "./constants";
+
+export function initializeHttpsServer(app: Express) {
+  const httpsServer = https.createServer(
+    {
+      key: fs.readFileSync(
+        path.join(__dirname, "../../../.cert/smile-dashboard-web-key.pem")
+      ),
+      cert: fs.readFileSync(
+        path.join(__dirname, "../../../.cert/smile-dashboard-web-cert.pem")
+      ),
```

Comment:
I'm leaning towards moving these to a props/constants instead. 

Maybe a const for the cert filepath and props for the key and certificate filenames.

Reply:
@alisman I'm curious what you think as well. This is a non-issue really and more of a preference. I don't think this should hold up merging the PR.

Reply:
I agree with making this not hard coded here. Will go ahead and merge this PR but won't delete it to give Aaron time to respond.